.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,128 @@
+name: Publish Docker Image
+
+on:
+  push:
+    branches:
+      - main
+      - v1.16
+    tags:
+      - v1.*
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      REPOSITORY: ${{ secrets.DOCKER_HUB_ORGS }}/fluentd-kubernetes-daemonset
```

Comment:
Do we need to make `DOCKER_HUB_ORGS` a secret variable?
Can't we just use a literal `fluent`?

Reply:
This is intended to test with forked repository.
Prepared GitHub workflow becomes stable enough, then we replace it.

Reply:
I see!